src/utils/api.js

```diff
@@ -31,3 +31,11 @@ export const fetchComments = (reviewId) => {
       return res.data.comments;
     });
 };
+
+export const patchVoteByReviewID = (reviewId, votes) => {
+  return axios
+    .patch(`https://sg-nc-games.onrender.com/api/reviews/${reviewId}`, {inc_votes: votes})
```

Comment:
I would love for you to implement a baseURL in your api.js file using axios.create - this will take some bulkiness out of all your functions in this file, see if you can make that happen